workflows/MODEL_TO_DATA.nf

```diff
@@ -46,13 +47,14 @@ workflow MODEL_TO_DATA {
     CREATE_FOLDERS(submission_ch, "create", params.project_name)
     UPDATE_SUBMISSION_STATUS_BEFORE_RUN(submission_ch, "EVALUATION_IN_PROGRESS")
     RUN_DOCKER(submission_ch, SYNAPSE_STAGE.output, params.cpus, params.memory, CREATE_FOLDERS.output, UPDATE_SUBMISSION_STATUS_BEFORE_RUN.output)
-    UPDATE_FOLDERS(submission_ch, params.project_name, RUN_DOCKER.output.map { it[1] }, RUN_DOCKER.output.map { it[2] })
-    // UPDATE_SUBMISSION_STATUS_AFTER_RUN(RUN_DOCKER.output.map { it[0] }, "ACCEPTED")
-    // VALIDATE(RUN_DOCKER.output, UPDATE_SUBMISSION_STATUS_AFTER_RUN.output, params.validation_script)
-    // UPDATE_SUBMISSION_STATUS_AFTER_VALIDATE(submission_ch, VALIDATE.output.map { it[2] })
-    // ANNOTATE_SUBMISSION_AFTER_VALIDATE(VALIDATE.output)
-    // SCORE(VALIDATE.output, UPDATE_SUBMISSION_STATUS_AFTER_VALIDATE.output, ANNOTATE_SUBMISSION_AFTER_VALIDATE.output, params.scoring_script)
-    // UPDATE_SUBMISSION_STATUS_AFTER_SCORE(submission_ch, SCORE.output.map { it[2] })
-    // ANNOTATE_SUBMISSION_AFTER_SCORE(SCORE.output)
-    // SEND_EMAIL(params.view_id, submission_ch, params.email_with_score, ANNOTATE_SUBMISSION_AFTER_SCORE.output)
+    renamed_file = RENAME_FILE(submission_ch, RUN_DOCKER.output.map { it[1] })
```

Comment:
Something interesting happened in one of my latest runs (5 submissions per submitter) where the predictions files were uploaded on behalf of another submission. See the command output for the UPDATE_FOLDERS task for submissions 9743264 and 9743264 for context: [this run](https://tower-dev.sagebionetworks.org/orgs/Sage-Bionetworks/workspaces/example-dev-project/watch/3NcCICcbOwTRzg)).

Technically this swap doesn't do any harm (the predictions files all end up in the same place at the end of the day), but it does call for attention the way the output values are fed into subsequent processes. My solution was to explicitly assign a variable to the output file and feeding that new variable (`renamed_file`) to `UPDATE_FOLDERS`, instead of implicitly dumping `RENAME_FILE.output` into `UPDATE_FOLDERS`.

Reply:
[New run with the fix](https://tower-dev.sagebionetworks.org/orgs/Sage-Bionetworks/workspaces/example-dev-project/watch/5T1584ulSs24so)

Reply:
This is very strange, were you able to determine why that was happening?

Reply:
Prior to this change, I was feeding `RENAME_FILE.output` to `UPDATE_FOLDERS` which from my understanding is basically the equivalent of telling Nextflow "just run `UPDATE_FOLDERS` with the first output channel of `RENAME_FILE` that you get".

After looking at the submit/start/stop times of the processes for submissions `9743264` and `9743265` in [the workflow where the swap happened](https://tower-dev.sagebionetworks.org/orgs/Sage-Bionetworks/workspaces/example-dev-project/watch/3NcCICcbOwTRzg), I'm seeing that `RENAME_FILE` for -65 finished 1 second faster than for -64:

`9743264`
<img width="287" alt="image" src="https://github.com/Sage-Bionetworks-Workflows/nf-synapse-challenge/assets/32107699/9465c209-8d33-449b-91ef-0b4995301fc2">

`9743265`
<img width="283" alt="image" src="https://github.com/Sage-Bionetworks-Workflows/nf-synapse-challenge/assets/32107699/4b3a249e-84fe-48ad-9c07-01211a9541a5">

This leads me to think that it was an issue of the first output channel for RENAME_FILE being the one that was grabbed for -64, and that happened to be the output path for -65's `predictions` file. But then it leaves me asking how this mixup doesn't happen for things like the `docker.log` files which don't seem to have any mixups (for my test1-5 submissions I have different stdouts that get written to the docker logs to verify this). I'm still trying to grasp the concept of channels.

For what it's worth: in the [new run](https://tower-dev.sagebionetworks.org/orgs/Sage-Bionetworks/workspaces/example-dev-project/watch/5T1584ulSs24so), RENAME_FILE for -65 doesn't start until -64 ends, which prevents this issue from happening. [Here](https://tower-dev.sagebionetworks.org/orgs/Sage-Bionetworks/workspaces/example-dev-project/watch/2RHAGSG6w8JlqD) is another run with the latest branch that ran as expected and shows the same behavior.